std/file.d

```diff
@@ -1405,6 +1405,13 @@ assert("/usr/share/include".isDir);
     }
 }
 
+@property bool isDir(R)(auto ref R name)
+if (!(isInputRange!R && isSomeChar!(ElementEncodingType!R))
+    && is(StringTypeOf!R))
+{
+    return isDir(cast(StringTypeOf!R)name);
```

Comment:
Thanks. It took me a while to spot the difference. It looks better and helps the compiler a bit, but the cast is also safe (at least, the compiler doesn't complain).